### PR TITLE
Fixes excessive bandwidth

### PR DESCRIPTION
**Pull request by 88plug**
Default seed_mode to false.

### CODE REVIEW

File: charts/akash-node/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ version: 11.1.0
 # Refs
 # https://github.com/akash-network/node/releases
 # https://github.com/akash-network/net/blob/main/mainnet/version.txt
-appVersion: 0.36.0
+appVersion: 0.36.1
```
Comment (ghost) [inline charts/akash-node/Chart.yaml:29]: There is no 0.36.1 node binary. Please keep it 0.36.0.

File: charts/akash-node/templates/statefulset.yaml [hunk 2]
```diff
@@ -93,7 +93,7 @@ spec:
             - name: AKASH_ROSETTA_ENABLE
               value: "false"
             - name: AKASH_P2P_SEED_MODE
-              value: "true"
+              value: "false"
```
Comment (ghost) [inline charts/akash-node/templates/statefulset.yaml:96]: This is not the solution the excessive bandwidth issue.  Please hold on and see the update in the issue.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@88plug Thank you for the PR, however this is not the solution the excessive bandwidth issue.  Please hold on and see the update in the issue https://github.com/akash-network/support/issues/231

**Comment by 88plug** [comment]
This should not be closed, updated version.

**Comment by ghost** [comment]
> This should not be closed, updated version.

Setting `AKASH_P2P_SEED_MODE`  to `false` isn't a solution to the "excessive bandwidth" issue.

**Pull request closed without merging**